kubemarine/procedures/check_paas.py

```diff
@@ -901,6 +904,14 @@ def container_runtime_configuration_check(cluster: KubernetesCluster) -> None:
                 cluster.log.debug(yaml.safe_dump(yaml.safe_load(diff.to_json())))
                 success = False
 
+            diff = DeepDiff(expected_registries, actual_registries.get(node.get_host(), {}))
```

Comment:
This is not aligned with the above check when we take actual config from nodes as the basis and compare it with the expected config from inventory. I think they should be aligned, but I have no strong position what should be the order.

`DeepDiff(actual_config, expected_config)` variant has semantics "what will be changed on cluster if --tasks cri.configure is called".

Reply:
fixed